os/drivers/gpio/gpio.c

```diff
@@ -583,6 +586,19 @@ static int gpio_close(FAR struct file *filep)
 		return ret;
 	}
 
+#ifndef CONFIG_DISABLE_POLL
+	/* Check if this file is registered in a list of waiters for polling.
```

Comment:
please give the space below and up from the comments.
it enhance the readability.